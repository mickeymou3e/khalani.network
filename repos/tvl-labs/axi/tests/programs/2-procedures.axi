../../tutorial/2-procedures.axi