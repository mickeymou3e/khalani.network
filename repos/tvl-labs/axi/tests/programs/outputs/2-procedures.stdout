Result: Nothing
Result: Nothing
Result: Nothing
Result: Nothing
Result: Nothing
Result: Proposition(and(and(a, b), c))
Result: Nothing
Result: Proved(if(a, if(b, if(c, and(and(a, b), c)))))
Result: Nothing
Result: Proposition(and(and(a, b), c))
Result: Nothing
Result: Proposition(and(and(a, b), c))
